jitsi-meet-run

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+EXTRA_ARGS=()
+
+declare -i JITSI_USE_WAYLAND="${JITSI_USE_WAYLAND:-0}"
+
+# Additional args for enabling Wayland
+if [[ "${JITSI_USE_WAYLAND}" -eq 1 && "${XDG_SESSION_TYPE}" == "wayland" ]]; then
+    EXTRA_ARGS+=(
+        "--ozone-platform=wayland"
+    )
+fi
+
+echo "Debug: Will run Jitsi Meet with the following arguments: ${EXTRA_ARGS[@]}"
+echo "Debug: Additionally, user gave: $@"
```

Comment:
These have to go, might be interesting to add an echo to the if-case though, something along the lines of;

```bash
echo "JITSI_USE_WAYLAND set, adding --ozone-platform=wayland to launch options"
```